jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -274,7 +274,7 @@ public APIContext getServletContext()
     @ManagedAttribute("Checks if the /context is not redirected to /context/")
     public boolean getAllowNullPathInfo()
     {
-        return _allowNullPathInfo;
+        return _coreContextHandler.getAllowNullPathInContext();
```

Comment:
Seems an unrelated change to the thrust of this PR, but as it's a good one, I'm gonna let it go through to the keeper ;)